### ISSUE TITLE
English language 

### ISSUE THREAD

**Issue by MrConqueror** [RyensX/MediaBox#49]
can this be change into English

**Comment by RyensX** [comment]
Of course, in fact the application has been prepared in advance to support multiple languages.
If you can translate, you can modify the file [here](https://github.com/RyensX/MediaBox/tree/dev/app/src/main/res/values) and submit Pull requests to provide English support.
However, plugins are not able to support multiple languages, this is because most of the data sources for which plugins exist today are Chinese.
